docs/root/configuration/http_filters/ext_authz_filter.rst

```diff
@@ -74,6 +74,29 @@ A sample filter configuration for a raw HTTP authorization server:
       hosts:
         - socket_address: { address: 127.0.0.1, port_value: 10003 }
 
+A sample virtual host and route filter configuration:
```

Comment:
This is great. I think we could make it even more explicit by adding a header just for per-route configuration. E.g, https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/buffer_filter#per-route-configuration.

Reply:
added the head line and expanded the text

Reply:
also added comments for the .proto